geoportal/c2cgeoportal_geoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -31,9 +31,14 @@ Changes to apply
    using Sass in place of Less. The major change is that the variables will be prefixed by '$'
    instead of '@' and we will use rem as recommended unit.
 
-3. Docker version: To be able to use the mapfile faster we don't use the mako anymore for that,
-   to convert them use:
+3. Docker version: To be able to deploy mapfile modifications faster we don't use the mako anymore for that.
+   1. To convert them use:
    `./docker-run make --always-make --makefile=CONST_convert2tmpl.mk to-tmpl`
+   If some files fail you can add more variable in the `vars_convert2tmpl.yaml` file,
+   and verify that the new variable is present in the config service as environment variable
+   (concerned files: `.env.mako` and `docker-compose.yaml`).
+   2. In the `vars` files you should also do the conversion present in the file `vars_convert2tmpl.yaml`
```

Comment:
2. You should adapt your `vars` files as illustrated in the file `vars_convert2tmpl.yaml`